main.py

```diff
@@ -0,0 +1,241 @@
+    print(f'Highest: {highest_temp["temperature"] or 0}C on {datetime.strptime(highest_temp["date"], "%Y-%m-%d").strftime("%B, %Y")}')
+
+    lowest_temp = calObj.calculateTemperature('L')
+    # print(lowest_temp["date"])
+    print(f'Lowest: {lowest_temp["temperature"] or 0}C on {datetime.strptime(lowest_temp["date"], "%Y-%m-%d").strftime("%B, %Y")}')
+
+    high_humidity = calObj.calculateTemperature('Hu')
+    # print(high_humidity["date"])
+    print(f'Humidity: {high_humidity["temperature"] or 0}% on {datetime.strptime(high_humidity["date"], "%Y-%m-%d").strftime("%B, %Y")}')
+
+def displayMonthData(year, month, bar_count):
+    files_data = LoadFileData(year, month)
+    calObj = CalculateTemperatureValues(files_data)
+    
+    avg_high_temp = calObj.calculateTemperature('avg_H')
+    print(f'Highest Average: {avg_high_temp["temperature"] or 0}C')
+    
+    avg_low_temp = calObj.calculateTemperature('avg_L')
+    print(f'Lowest Average: {avg_low_temp["temperature"] or 0}C')
+        
+    avg_mean_humidity = calObj.calculateTemperature('avg_Hu')
+    print(f'Average Mean Humidity: {avg_mean_humidity["temperature"] or 0}%')
+    displayReport(calObj, bar_count = bar_count)
+    
+def displayReport(calObj, bar_count = 2):
+    print(f'\n{months[calObj.data.month]} {calObj.data.year}')
+    swap = True
+    display_record = {}
+    for value in calObj.getSingleDateRecord():
+        splitted_date  = value and value["date"].split('-')
+        if(not splitted_date): 
+            swap = not swap
+            continue
+            
+        display_record["date"] = splitted_date[2]
+        temperature = value and value["value"]
+        display_record[(swap and 'max_temp') or 'min_temp'] = temperature
+        signs = ''
+        
+        for value in range(int(display_record[(swap and 'max_temp') or 'min_temp'])):
+            if(swap):
+                signs += f'{RED}+'
+            else:
+                signs += f'{BLUE}+'
+                
+        display_record[(swap and 'max_temp_sign') or 'min_temp_sign'] = signs
+        if bar_count < 2:
+            if(not swap): print(f'{display_record["date"]} {display_record["min_temp_sign"]}{display_record["max_temp_sign"]}{RESET} {display_record["min_temp"]}-{display_record["max_temp"]}')
+        else: 
+            if(swap):
+                print(f'{display_record["date"]} {display_record["max_temp_sign"]}{RESET} {display_record["max_temp"]}')
+            else:
+                print(f'{display_record["date"]} {display_record["min_temp_sign"]}{RESET} {display_record["min_temp"]}')
+                
+        swap = not swap
+    
+def main():
+    
+    file_dates = input("Please Enter Year and Month: ")
+    splitted_files_date = file_dates.split(' ')
+    
+    for file_date in splitted_files_date:
+        splitted_file_date = file_date.split('/')
+        if len(splitted_file_date) >= 2:
+            bar_count = int(input("Please Enter Number of Report bar_counts 1-2: "))
```

Comment:
What is the purpose of this?

Reply:
it is used to display report in one horizontal bar or 2 horizontal bars

Reply:
Lets use [Command Line Arguments](https://www.geeksforgeeks.org/command-line-arguments-in-python/) as in examples mentioned in task?